src/CompanyService.Data.Provider.MsSql.Ef/Migrations/20210817223800_AddWorkDaysApiUrlColumn.cs

```diff
@@ -0,0 +1,19 @@
+﻿using LT.DigitalOffice.CompanyService.Models.Db;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace LT.DigitalOffice.CompanyService.Data.Provider.MsSql.Ef.Migrations
+{
+    [DbContext(typeof(CompanyServiceDbContext))]
+    [Migration("20210817223800_AddWorkDaysApiUrlColumn")]
+    public class _20210817223800_AddWorkDaysApiUrlColumn : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<string>(
+            name: "WorkDaysApiUrl",
```

Comment:
use nameof(DbCompany.WorkDaysApiUrl)

Reply:
is it better than simply write a name? Why?

Reply:
Since it is possible to make a typo in such a line and not notice it, in the case of nameof you will catch an error.

Reply:
This is the same as with DbCompany.TableName, it is easier to write table: "tableName", but we don't.

Reply:
okay
i dont like one thing here - if we update DbCompany model in the future by deleting property workdaysapiurl, will we have a compilation error in this migration? or it is not important?

Reply:
you will have a compilation error, or if you just rename it, you will have an exception when performing the migration, because in previous migrations there would have been a different column name, but you are waiting for the old name